<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="AG-5CG4493YSW" failures="2" tests="2" name="Test(failed)" time="70.819" errors="0" timestamp="2025-08-01T14:45:37 IST">
  <testcase classname="testCases.TC002_LoginTest" name="verify_login" time="31.433">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC002_LoginTest.verify_login(TC002_LoginTest.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_account_registration" time="31.417">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC001_AccountRegistrationTest.verify_account_registration(TC001_AccountRegistrationTest.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_account_registration -->
</testsuite> <!-- Test(failed) -->
